scripts/sql/provisioning/2-provision-database.sql

```diff
@@ -0,0 +1,311 @@
+CREATE SEQUENCE IF NOT EXISTS PUBLIC.BOARD_SEQUENCE START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.ATTRIBUTE_DATA_TYPE_ATTRIBUTE_DATA_TYPE_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.BB_CODE_ATTRIBUTE_BB_CODE_ATTRIBUTE_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.BB_CODE_ATTRIBUTE_MODE_BB_CODE_ATTRIBUTE_MODE_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.BB_CODE_CONFIG_BB_CODE_CONFIG_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.BOARD_BOARD_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.BR_BOARD_PERMISSION_BR_BOARD_PERMISSION_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.BR_USER_PERMISSION_BR_USER_PERMISSION_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.EMAIL_ADDRESS_EMAIL_ADDRESS_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.FORUM_FORUM_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.MESSAGE_HISTORY_MESSAGE_HISTORY_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.MESSAGE_MESSAGE_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.PERMISSION_PERMISSION_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.THREAD_THREAD_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE SEQUENCE IF NOT EXISTS ZFGBB.USER_USER_ID_SEQ AS INTEGER START
+WITH
+	1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+
+CREATE TABLE IF NOT EXISTS ZFGBB.IP_ADDRESS (
+	IP_ADDRESS_ID INTEGER NOT NULL,
+	IP TEXT NOT NULL,
+	IP_V6_FLAG BOOLEAN NOT NULL,
+	IS_SPAMMER_FLAG BOOLEAN NOT NULL,
+	CREATED_TS TIMESTAMP WITHOUT TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	UPDATED_TS TIMESTAMP WITHOUT TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	CONSTRAINT IP_ADDRESS_PKEY PRIMARY KEY (IP_ADDRESS_ID)
+);
+
+ALTER TABLE IF EXISTS ZFGBB.IP_ADDRESS OWNER TO ZFGCADMIN;
+
+CREATE TABLE IF NOT EXISTS ZFGBB.ATTRIBUTE_DATA_TYPE (
+	ATTRIBUTE_DATA_TYPE_ID INTEGER GENERATED BY default AS PRIMARY KEY,

```

Comment:
you can use this INTEGER GENERATED BY DEFAULT AS PRIMARY KEY definition on all of the primary keys - some of them are using DEFAULT NEXTVAL